src/ng/parse.js

```diff
@@ -401,7 +400,7 @@ AST.prototype = {
 
   program: function() {
     var body = [];
-    while (true) {
+    for (;;) {
```

Comment:
I could perhaps disable the checkLoops option:
http://eslint.org/docs/rules/no-constant-condition#checkloops

On Fri, Aug 5, 2016 at 4:31 PM, Martin Staffa <notifications@github.com>
wrote:

> In src/ng/parse.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r73701186>:
>
> > @@ -401,7 +400,7 @@ AST.prototype = {
> >
> >    program: function() {
> >      var body = [];
> > -    while (true) {
> > +    for (;;) {
>
> This is a bit unusual (at least to me) and could be explained with a
> comment. Or the no-constant-condition rule could be disabled for this line
> (while keeping while(true)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.js/pull/14952/files/4ab2433bb71fe2bf692ca169c46363385a3806e4#r73701186>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnjmiORxRvtOdyVjgvr2NaYpP_954ks5qc0lbgaJpZM4JU90D>
> .
>



-- 
Michał Gołębiowski